chemprop/uncertainty/uncertainty_evaluator.py

```diff
@@ -161,20 +182,23 @@ def evaluate(
         targets: List[List[float]],
         preds: List[List[float]],
         uncertainties: List[List[float]],
+        mask: List[List[bool]],
     ):
         targets = np.array(targets, dtype=int)  # shape(data, tasks)
+        mask = np.array(mask, dtype=bool)
         uncertainties = np.array(uncertainties)
         preds = np.array(preds)
-        nll = np.zeros_like(targets)
+        nll = []
         for i in range(targets.shape[1]):
             task_preds = uncertainties[:, i]
             task_targets = targets[:, i]  # shape(data)
+            task_mask = mask[:, i]
             bin_targets = np.zeros_like(preds[:, 0, :])  # shape(data, classes)
             bin_targets[np.arange(targets.shape[0]), task_targets] = 1
             task_likelihood = np.sum(bin_targets * task_preds, axis=1)
             task_nll = -1 * np.log(task_likelihood)
```

Comment:
```suggestion
            task_mask = mask[:, i]
            task_preds = uncertainties[task_mask, i]
            task_targets = targets[task_mask, i]  # shape(data)
            bin_targets = np.zeros_like(task_targets)  # shape(data, classes)
            bin_targets[np.arange(task_targets.shape[0]), task_targets] = 1
            task_likelihood = np.sum(bin_targets * task_preds, axis=1)
            task_nll = -1 * np.log(task_likelihood)
```
I think this is the similar one. I am not sure if this suggestion is correct, please correct me if I am wrong.

Reply:
You are right. This was a mistake, the mask was added but not actually used anywhere.

Reply:
Fixed now